server/server_test.go

```diff
@@ -1922,6 +1922,13 @@ func getNetwork(color string, tenant string) map[string]interface{} {
 	}
 }
 
+func getSchema(schemaID string) *schema.Schema {
```

Comment:
Why isn't this function defined in the same file it's used?

Reply:
moved